src/function/table/system/duckdb_databases.cpp

```diff
@@ -29,6 +29,9 @@ static unique_ptr<FunctionData> DuckDBDatabasesBind(ClientContext &context, Tabl
 	names.emplace_back("internal");
 	return_types.emplace_back(LogicalType::BOOLEAN);
 
+	names.emplace_back("readonly");
```

Comment:
Could we add it as the last element instead?

Reply:
Done.